lib/screens/transfer/send/send_confirmation/interactor/viewmodels/send_confirmation_arguments.dart

```diff
@@ -1,11 +1,9 @@
+import 'package:seeds/datasource/local/models/eos_transaction.dart';
+
 class SendConfirmationArguments {
-  final String account;
-  final String name;
-  final Map<String, dynamic> data;
+  final EOSTransaction transaction;
```

Comment:
Same here not sure if a class is worth having for one filed instead of using that type directly maybe for readably

Reply:
Hmm I am fine with having a separate arguments class